spec/requests/api/v1/suggestions_spec.rb

```diff
@@ -13,13 +13,12 @@
       get '/api/v1/suggestions', headers: headers, params: params
     end
 
-    let(:bob)    { Fabricate(:account) }
-    let(:jeff)   { Fabricate(:account) }
+    let(:bob) { Fabricate(:account) }
+    let(:jeff) { Fabricate(:account) }
     let(:params) { {} }
 
     before do
-      PotentialFriendshipTracker.record(user.account_id, bob.id, :reblog)
-      PotentialFriendshipTracker.record(user.account_id, jeff.id, :favourite)
+      Setting.bootstrap_timeline_accounts = [bob, jeff].map(&:acct).join(',')
```

Comment:
I think this needs to be restored after the example runs.

Reply:
Are you sure? This isn't Rails configuration. It goes to the settings table, which is part of the wipe between runs.

Reply:
I think you may be right, we have a pattern of saving and restoring them in specs, but it doesn't seem it's needed (anymore?), I'll take care of that in a separate pull request.